dagstore_control.go

```diff
@@ -41,6 +42,15 @@ func (d *DAGStore) control() {
 	var wFailure = &waiter{ctx: d.ctx, outCh: d.failureCh}
 
 	for {
+		// do we need to do a GC ?
+		ts, err := d.transientDirSize()
```

Comment:
`ts` usually stands for timestamp.

```suggestion
		dirSize, err := d.transientDirSize()
```